### PR TITLE
Add callstats support for mobile

### PR DESCRIPTION
**Pull request by saghul**
See https://github.com/jitsi/lib-jitsi-meet/pull/448

### CODE REVIEW

**Comment by saghul** [comment]
Same as https://github.com/jitsi/lib-jitsi-meet/pull/448, this is waiting on the new react-native-callstats package.

**Comment by saghul** [comment]
@lyubomir You may take the first patch and push it as-is, it Just Works but I ran into it while working with callstats, that's why it's part of this PR.

**Comment by saghul** [comment]
Rebased after latest changes in config storage.

File: react/features/base/config/reducer.js [hunk 1]
```diff
@@ -18,15 +18,7 @@ const INITIAL_STATE = {
...
     // fastest to merely disable them.
-    disableAudioLevels: true,
-
-    // FIXME Lib-jitsi-meet uses HTML script elements to asynchronously load
-    // certain pieces of JavaScript. Unfortunately, the technique doesn't work
-    // on React Native (because there are no HTML elements in the first place).
-    // Fortunately, these pieces of JavaScript currently involve third parties
-    // and we can temporarily disable them (until we implement an alternative to
-    // async script elements on React Native).
-    disableThirdPartyRequests: true
```
Comment (paweldomas) [inline react/features/base/config/reducer.js]: It still lives here: https://github.com/saghul/lib-jitsi-meet/blob/892d318f1dee96b772796b1f0c290e01d52ad1f9/modules/statistics/statistics.js#L106

**Comment by lyubomir** [comment]
@paweldomas & @saghul, here's an update from today and a request or two that I'd like to ask you to please address on your Tuesday:

* I merged the first commit of this PR into master i.e. "deps: use latest jsSHA".

* Pawel created in jitsi-meet a branch rn-callstats which is a rebase of this PR. As I and Pawel are developers of the jitsi-meet project and Saul hasn't changed the defaults on this PR, I and Pawel are able to push into Saul's branch. Which I did and now the branch rn-callstats is the same as this PR at the time of this writing. Maybe we can delete the branch rn-callstats and continue here. Whatever you decide but one is obsolete.

* I merged https://github.com/jitsi/lib-jitsi-meet/pull/448 into master. Pawel had it rebased in a branch rn-callstats in lib-jitsi-meet so I used Pawel's rebased commits. I don't see negative effects from the merge because it's basically dormant until we merge this PR.

* The branch rn-callstats in lib-jitsi-meet contains 2 commits from Pawel now. I commented on 1 of the commits because I think it shouldn't be merged and I mentioned both your names - please work together to have a resolution for these 2 commits/Pawel's rn-callstats branch.

* In both lib-jitsi-meet and jitsi-meet we're now using react-native-callstats 3.19.8 while 3.23.0 is available. I guess the former was the latest version available when Saul created the PRs but now that we're merging doesn't it make sense to do all the testing with the latest version today?

**Comment by saghul** [comment]
Superseded by #1941

**Pull request closed without merging**